script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -137,61 +137,62 @@ I haven't forgotten a single thing you said, actually!"
 [sha:6d83565214320233e63cea4ec9e4f5e6e4d8a0bf]{
 -- Page 42, Offset 14908.
 -- 正門前という事を忘れて先輩にくってかかる。
-I lash out at her, forgetting that I'm right in front of the school entrance.
+I lash out at her, forgetting that we're right in front of the school entrance.
 }
 [sha:2fa66e70b9b80eb7835d2e01c4e2541af734952c]{
 -- Page 42, Offset 14909.
 -- 自分でもなんでこんなに気が動転しているのか分からない。
-I have a hard time understanding why I've lost my composure like this.
+I'm not entirely sure why I've lost my composure this badly.
```

Comment:
I lost my

Reply:
I feel like it's still lost here? `I have lost` feels more continuous than `I lost` to me

Reply:
maybe it's just a colloquialism but "I have lost" sounds quite a lot less natural to me